diff --git a/base/BUILD.gn b/base/BUILD.gn
index 539782c0683e197194357296f5ba8eb86b0cfa65..49329fb2f30f2c1b5998cdf3c30f0f5c6dd0980e 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5049,6 +5049,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5723,6 +5724,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
