diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d3ed86a7c88323e458aeb62737d9cc59ad4c673a..ba3e80234e5cf6aac3b2e232aaf26d72f9658a9f 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4656,6 +4656,7 @@ const FeatureEntry::FeatureVariation kSplitViewHorizontalVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
