diff --git a/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc b/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc
index e01b131da9a76d1432c7766f0419c82128c79087..e2bb9c286848df266abdbaf4f54a4644096ff264 100644
--- a/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc
+++ b/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc
@@ -244,6 +244,7 @@ void ExtensionsMenuMainPageView::CreateAndInsertMenuEntry(
     ExtensionsMenuViewModel::MenuEntryState entry_state,
     int index) {
   auto extension_id = action_model->GetId();
+  BRAVE_EXTENSION_MENU_MAIN_PAGE_VIEW_CREATE_AND_INSERT_MENU_ITEM
   // base::Unretained() below is safe because `menu_handler_` lifetime is
   // tied to this view lifetime by the extensions menu coordinator.
   auto item = std::make_unique<ExtensionsMenuEntryView>(
