diff --git a/chrome/installer/linux/common/installer.py b/chrome/installer/linux/common/installer.py
index 998f187535b5974153210ac5a1268d26ceace15f..8777fdee7116622654c571316c2130698613f2f1 100644
--- a/chrome/installer/linux/common/installer.py
+++ b/chrome/installer/linux/common/installer.py
@@ -199,6 +199,8 @@ def gen_changelog(config: "InstallerConfig",
     gzlog_dir = (
         config.staging_dir /
         f"usr/share/doc/{config.info_vars['PACKAGE']}-{config.channel}")
+    if config.channel == "stable":
+        gzlog_dir = (config.staging_dir / f"usr/share/doc/{config.info_vars['PACKAGE']}")
     gzlog_dir.mkdir(parents=True, exist_ok=True)
     gzlog = gzlog_dir / "changelog.gz"
 
@@ -335,6 +337,7 @@ class InstallerConfig:
     uri_scheme: str = ""
     extra_desktop_entries: str = ""
     startup_wm_class: str = ""
+    package_and_channel: str = ""
 
     @classmethod
     def from_args(cls, args: argparse.Namespace,
@@ -695,6 +698,7 @@ class InstallerConfig:
                 icon_suffix = "_dev"
             elif self.channel == "canary":
                 icon_suffix = "_canary"
+        icon_suffix = {"beta": "_beta", "dev": "_dev", "nightly": "_nightly", "unstable": "_dev"}.get(self.channel, "") if self.branding in ("brave", "brave_origin") else ""
 
         icon_sizes = [16, 24, 32, 48, 64, 128, 256]
         logo_resources_png = []
@@ -1173,3 +1177,4 @@ class Installer:
                         msg += f", but they were {oct(actual_perms)}"
                         print(msg, file=sys.stderr)
                         sys.exit(1)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
