diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
index c646774c2fdea2527ed6825d49c8dcb83c11b709..e2580854e90238ccf10be6671fb5c8995a4edd1e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
@@ -83,7 +83,8 @@ class ChromeContextMenuItem {
         Item.INSPECT_ELEMENT,
         Item.COPY_VIDEO_FRAME,
         Item.DOWNLOAD_VIDEO_FRAME,
-        Item.LENS_OVERLAY
+        Item.LENS_OVERLAY,
+        Item.COPY_CLEAN_LINK,
     })
     @Retention(RetentionPolicy.SOURCE)
     public @interface Item {
@@ -147,8 +148,9 @@ class ChromeContextMenuItem {
         int COPY_VIDEO_FRAME = 46;
         int DOWNLOAD_VIDEO_FRAME = 47;
         int LENS_OVERLAY = 48;
+        int COPY_CLEAN_LINK = 49;
         // ALWAYS UPDATE!
-        int NUM_ENTRIES = 49;
+        int NUM_ENTRIES = COPY_CLEAN_LINK + 1;
     }
 
     /** Mapping from {@link Item} to the ID found in the ids.xml. */
@@ -202,6 +204,7 @@ class ChromeContextMenuItem {
         R.id.contextmenu_copy_video_frame, // Item.COPY_VIDEO_FRAME
         R.id.contextmenu_download_video_frame, // Item.DOWNLOAD_VIDEO_FRAME
         R.id.contextmenu_lens_overlay, // Item.LENS_OVERLAY
+        R.id.contextmenu_copy_clean_link, // Item.COPY_CLEAN_LINK
     };
 
     /** Mapping from {@link Item} to the ID of the string that describes the action of the item. */
@@ -255,6 +258,7 @@ class ChromeContextMenuItem {
         R.string.contextmenu_copy_video_frame, // Item.COPY_VIDEO_FRAME
         R.string.contextmenu_download_video_frame, // Item.DOWNLOAD_VIDEO_FRAME
         R.string.contextmenu_search_tab_with_google_lens, // Item.LENS_OVERLAY
+        R.string.contextmenu_copy_clean_link, // Item.COPY_CLEAN_LINK
     };
 
     /**
