diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 0bef8ea3218b6f10287cbfb79a1e63f27d15d9b0..588918086f23c4502a58c8ab54fba566455c2548 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1253,6 +1253,7 @@ static_library("ui") {
     # split out yet.
     "//chrome/browser/webauthn:impl",
   ]
+  import("//brave/browser/sources.gni") import("//brave/components/brave_wallet/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from + brave_chrome_browser_ui_deps allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -5564,6 +5565,7 @@ group("ui_public_dependencies") {
     "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags",
     "//components/safe_browsing:buildflags",
+    "//components/saved_tab_groups/public",
     "//components/segmentation_platform/public",
     "//components/sync",
     "//components/sync_user_events",
@@ -5576,6 +5578,7 @@ group("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
