diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
index 6c95314c2154fd5fab1ba5d4cea4b8d02000b4d9..3c52e615eb6b3f5a18da2bdc8ae7e4f8afadbec1 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
@@ -715,6 +715,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
 #else
   info.incognito_access_pending_change = false;
 #endif
+  BRAVE_CREATE_EXTENSION_INFO_HELPER
 
   // User Scripts toggle.
   info.user_scripts_access.is_enabled =
