diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
index 329a744e25d4d362f6b33a4b071e219ed9352406..22a4d84566a9c9c1176f6cddcd15e980309a4015 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
@@ -33,7 +33,7 @@ class TabStripControlButton;
 
 // Container for the tabstrip and the other views sharing space with it -
 // with the exception of the caption buttons.
-class HorizontalTabStripRegionView final : public TabStripRegionView {
+class HorizontalTabStripRegionView : public TabStripRegionView {
   METADATA_HEADER(HorizontalTabStripRegionView, TabStripRegionView)
 
  public:
