diff --git a/chrome/browser/resources/settings/site_settings/site_settings_util.ts b/chrome/browser/resources/settings/site_settings/site_settings_util.ts
index bb4f9d3411bee8249a113db6010d3a0426e42830..e153f2cf55157dd7107589b7089ac20e2b5d5670 100644
--- a/chrome/browser/resources/settings/site_settings/site_settings_util.ts
+++ b/chrome/browser/resources/settings/site_settings/site_settings_util.ts
@@ -110,7 +110,7 @@ export function getLocalizationStringForContentType(
     case ContentSettingsTypes.SMART_CARD_READERS:
       return null;
     default:
-      assertNotReached();
+      return null;
   }
 }
 
