diff --git a/chrome/browser/ui/startup/startup_browser_creator_impl.cc b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
index e650e818e87c66913a3372549ee4c0a593498dfb..34f118c917a6ce177de4acdd167f31f7d140564d 100644
--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -397,6 +397,7 @@ Browser* StartupBrowserCreatorImpl::OpenTabsInBrowser(
       params.disposition = WindowOpenDisposition::CURRENT_TAB;
       params.tabstrip_add_types = ADD_NONE;
       first_tab = false;
+      BraveModifyStartupTabNavigationParams(tab, browser, params);
       Navigate(&params);
       continue;
     }
@@ -427,6 +428,7 @@ Browser* StartupBrowserCreatorImpl::OpenTabsInBrowser(
     }
 #endif  // BUILDFLAG(ENABLE_RLZ)
 
+    BraveModifyStartupTabNavigationParams(tab, browser, params);
     Navigate(&params);
     first_tab = false;
   }
