diff --git a/chrome/browser/sessions/BUILD.gn b/chrome/browser/sessions/BUILD.gn
index 5d427fa243af8206b8cac29c960be2ff471d651e..e957cd85c82e509fc89755cb63fa2c4d26957ba2 100644
--- a/chrome/browser/sessions/BUILD.gn
+++ b/chrome/browser/sessions/BUILD.gn
@@ -165,6 +165,7 @@ source_set("impl") {
         "//chromeos/components/kiosk",
       ]
     }
+    import("//brave/chromium_src/chrome/browser/sessions/sources.gni") deps += brave_chromium_src_chrome_browser_sessions_deps
   }
 }
 
