diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index 1309a8a227416a8d36d54f0cce6a8a4ee96d5d37..73025ba8cb6a52bbf15eb5394a85c8cb6b20c233 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -144,6 +144,7 @@ android_resources("java_resources") {
     "java/res/xml/tab_archive_time_delta_preference.xml",
     "java/res/xml/tabs_settings.xml",
   ]
+  sources += brave_android_features_java_resources
   deps = [
     ":java_strings_grd",
     "//chrome/android:chrome_app_java_resources",
