diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index 00981ad79bd6ef6dc3159181950ae8ca7def36ad..4c7fba93b3a0d3e6b7108a718c560d241bef18a7 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -600,6 +600,7 @@ void ProfileImpl::TakePrefsFromStartupData() {
       CreateExtensionPrefStore(this, /*incognito_pref_store=*/false));
 
   ProfileKeyStartupAccessor::GetInstance()->Reset();
+  BRAVE_PROFILE_IMPL_TAKE_PREFS_FROM_STARTUP_DATA
 }
 #endif
 
