diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index d7641c08da15511290d9f4f974b2b4ecd6b041cd..b7db812add1071153835ae1e6d94c809bf884391 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -478,7 +478,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
       return;
     }
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), request_id, options, url_request,
+        std::move(receiver), request_id, options, BRAVE_CREATE_LOADER_AND_START,
         std::move(client), traffic_annotation);
   }
 
