diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 1d52e64e59bd256d9b2aeac11f71bd51d589aaa6..ce3af15e6f28ae4198cf6166f58bd430a0f0e09e 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/extensions/resources.gni")
 import("//build/config/chromebox_for_meetings/buildflags.gni")
 import("//build/config/features.gni")
 import("//build/config/ozone.gni")
@@ -646,6 +647,7 @@ source_set("extensions") {
     "//ui/native_window_tracker",
     "//ui/shell_dialogs",
   ]
+  deps += brave_extensions_resources
   public_deps = [
     "//chrome/browser/extensions/api/messaging",
     "//chrome/browser/media/webrtc",
