diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index 2c430b033368ec4fb93d49c8b36bd95cfe2d4def..9062febdac48b98c34d026c47d5e7615e102a784 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -186,6 +186,7 @@ source_set("internal") {
       "//chrome/browser/ui:browser_select_file_dialog_controller",
     ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
