diff --git a/chrome/browser/sessions/session_restore.cc b/chrome/browser/sessions/session_restore.cc
index 25d9ba2e1e50cc84164a14c4b6b550e174ae7f72..6f80874a961bb5beecf21fb6bbc82a8ee410e6db 100644
--- a/chrome/browser/sessions/session_restore.cc
+++ b/chrome/browser/sessions/session_restore.cc
@@ -1309,6 +1309,7 @@ class SessionRestoreImpl : public BrowserCollectionObserver {
                                : WindowOpenDisposition::NEW_BACKGROUND_TAB;
       params.tabstrip_add_types = add_types;
       is_first_tab = false;
+      BraveModifyStartupTabNavigationParams(startup_tab, browser, params);
       Navigate(&params);
     }
   }
