diff --git a/chrome/browser/ui/views/side_panel/side_panel.cc b/chrome/browser/ui/views/side_panel/side_panel.cc
index 478d1e0c5d887172e52191e8e93092c40193f239..56eaaf05058659273ebc6668f4d6bbdd297a3362 100644
--- a/chrome/browser/ui/views/side_panel/side_panel.cc
+++ b/chrome/browser/ui/views/side_panel/side_panel.cc
@@ -219,6 +219,7 @@ class ContentParentView : public views::View, public views::ViewObserver {
   }
 
   gfx::RoundedCornersF GetRoundedCorners() {
+    return brave::GetPanelContentsRoundedCorners(browser_view_);
     ChromeDistanceMetric corner_radius =
         ChromeDistanceMetric::DISTANCE_SIDE_PANEL_CONTENT_RADIUS;
     return GetLayoutProvider()
