diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index e9a7176ec1ef2771d8e9cdffc25bce2f760b298d..6ec3bc43b83500561a13556add7dd1db820e628a 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -198,7 +198,7 @@ class TabInterface;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
