diff --git a/chrome/browser/ui/views/tabs/vertical/vertical_tab_strip_view.cc b/chrome/browser/ui/views/tabs/vertical/vertical_tab_strip_view.cc
index d3a886fa9da2f45f542ceab6837882358dd2657e..0e000300a6a400b8a68855f8e9048d75bbe51be0 100644
--- a/chrome/browser/ui/views/tabs/vertical/vertical_tab_strip_view.cc
+++ b/chrome/browser/ui/views/tabs/vertical/vertical_tab_strip_view.cc
@@ -326,6 +326,8 @@ void VerticalTabStripView::OnChildMoved(TabCollectionNode* moved_node) {
 
   if (is_active_tab || has_focus) {
     switch (moved_node->type()) {
+      case TabCollectionNode::Type::TREE_NODE:
+        break;
       case TabCollectionNode::Type::TAB: {
         const tabs::TabInterface* tab =
             std::get<const tabs::TabInterface*>(moved_node->GetNodeData());
