diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
index f1505522bf599ada1707dbfc6eb59ac75defadac..fd81a0ee0edd58575081e0698a9572033a717104 100644
--- a/chrome/browser/ui/views/side_panel/BUILD.gn
+++ b/chrome/browser/ui/views/side_panel/BUILD.gn
@@ -176,6 +176,7 @@ source_set("side_panel") {
     "//ui/webui",
   ]
 
+  import("//brave/browser/ui/views/side_panel/sources.gni") deps += brave_browser_ui_views_side_panel_deps public_deps += brave_browser_ui_views_side_panel_public_deps
   if (enable_extensions) {
     public_deps += [
       "//extensions/browser",
