diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
index fa5db68ccf3a4264c6af2959d3b497477c6c16d9..2b9b650e37ad652316bf090698a49d31dd67e7e7 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
@@ -182,6 +182,7 @@ void PermissionPromptBubbleBaseView::CreatePermissionButtons(
     SetButtonStyle(ui::mojom::DialogButton::kOk, ui::ButtonStyle::kTonal);
     SetButtonStyle(ui::mojom::DialogButton::kCancel, ui::ButtonStyle::kTonal);
   }
+  BRAVE_PERMISSION_PROMPT_BUBBLE_BASE_VIEW
 }
 
 void PermissionPromptBubbleBaseView::CreateExtraTextLabel(
