diff --git a/chrome/browser/resources/history/BUILD.gn b/chrome/browser/resources/history/BUILD.gn
index d4c41286d6ef118ef6731bc90bfb519005755001..16a152cf9668c777c6e60d84926f9ec6426f8721 100644
--- a/chrome/browser/resources/history/BUILD.gn
+++ b/chrome/browser/resources/history/BUILD.gn
@@ -107,4 +107,5 @@ build_webui("build") {
     optimize_webui_host = "history"
     optimize_webui_in_files = [ "history.js" ]
   }
+  import("//brave/browser/resources/history/sources.gni") mojo_files = brave_history_mojo_files mojo_files_deps = brave_history_mojo_files_deps ts_files += brave_history_ts_files exclude_ts_preprocess_files = brave_history_ts_files preprocess_deps = brave_history_preprocess_deps
 }
