diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 07113c415743ed07ae2e9e4aa8dcf6168f9d0582..c35d0b4c5a7edf6f2246c2bb84f08276fd6f1166 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -5562,6 +5562,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
   // SiteInstance is created in a new BrowsingInstance.
   scoped_refptr<SiteInstance> site_instance;
   if (params.opener_suppressed) {
+    BRAVE_WEB_CONTENTS_IMPL_CREATE_NEW_WINDOW_OPENER_SUPPRESSED
     if (is_guest) {
       // For guests, noopener windows can be created in a new BrowsingInstance
       // as long as they preserve the guest's StoragePartition.
@@ -5659,7 +5660,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
