diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 6ce499c2931af931340b250716aa5eb7bb6e6e77..be28fa6ca9591b4206c754282fe0ed4c171f9c5c 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3840,6 +3840,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
