diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index b01f122afb1e8607d2b7eb2174ee936303fa6058..4e35f555e9108ea814575841eec050c4c84d68c1 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4886,6 +4886,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
