diff --git a/content/browser/renderer_host/navigation_controller_impl.cc b/content/browser/renderer_host/navigation_controller_impl.cc
index c44c25d8ac6d0f48d60d00de21411b5a2de13318..9cdcd5c92c5578a5d44d6406e4aa2dd5bd563c47 100644
--- a/content/browser/renderer_host/navigation_controller_impl.cc
+++ b/content/browser/renderer_host/navigation_controller_impl.cc
@@ -459,6 +459,7 @@ void RewriteUrlForNavigation(const GURL& original_url,
   *url_to_load = *virtual_url = original_url;
   BrowserURLHandlerImpl::GetInstance()->RewriteURLIfNecessary(
       url_to_load, browser_context, reverse_on_redirect);
+  MaybeRewriteVirtualURL(virtual_url);
 }
 
 #if DCHECK_IS_ON()
