diff --git a/DEPS b/DEPS
index c80d67dd656744003227f318459206a86f16615e..6c2ce0079a185ee0c5d3bbfe6ec73f27208ea261 100644
--- a/DEPS
+++ b/DEPS
@@ -4375,7 +4375,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
