diff --git a/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm b/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm
index 71b3dca8ed89210f224ef88799095a3d790b8d60..aab6db1a4018fcaa7d9eec103616934c6d729369 100644
--- a/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm
+++ b/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm
@@ -57,7 +57,7 @@ class SyncServiceFactoryTest : public PlatformTest {
  protected:
   // Returns the collection of default datatypes.
   syncer::DataTypeSet DefaultDatatypes() {
-    static_assert(63 == syncer::GetNumDataTypes(),
+    static_assert(63 + 1 == syncer::GetNumDataTypes(),
                   "When adding a new type, you probably want to add it here as "
                   "well (assuming it is already enabled).");
 
