diff --git a/ios/chrome/browser/sync/model/sync_service_factory.mm b/ios/chrome/browser/sync/model/sync_service_factory.mm
index d3ac7cb0ec8aba6e2bc705069c7b997e4c7fefb4..47e2b512360a57ba48e3533512e53b98f005d2b7 100644
--- a/ios/chrome/browser/sync/model/sync_service_factory.mm
+++ b/ios/chrome/browser/sync/model/sync_service_factory.mm
@@ -224,7 +224,7 @@ std::unique_ptr<KeyedService> BuildSyncService(ProfileIOS* profile) {
   init_params.debug_identifier = profile->GetProfileName();
   init_params.os_crypt_async = GetApplicationContext()->GetOSCryptAsync();
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   password_manager::PasswordReceiverService* password_receiver_service =
