diff --git a/third_party/android_deps/autorolled/BUILD.gn b/third_party/android_deps/autorolled/BUILD.gn
index 1686757c1e2615d636b8007bcd03ef1683e36940..8b773638fa77cb6e5cb684358b289ecefd134aa0 100644
--- a/third_party/android_deps/autorolled/BUILD.gn
+++ b/third_party/android_deps/autorolled/BUILD.gn
@@ -1036,6 +1036,7 @@ template("define_autorolled_targets") {
           "$google_play_services_package:google_firebase_firebase_encoders_json_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
         ]
+        import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
       }
     }
 
@@ -1061,6 +1062,7 @@ template("define_autorolled_targets") {
           "//third_party/android_deps:javax_inject_javax_inject_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
         ]
+        import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
       }
     }
 
