diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index ff021b45a53f6b1d7e7f4240e631de823df46587..9e26e5fd932c58ceae0f0328c81ca00711db44ec 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -382,6 +382,7 @@ class CORE_EXPORT HTMLCanvasElement final
   // Called when AxObject is created or its ignored state is changed.
   void OnAxObjectIgnoredStateChanged(bool is_ignored);
 
+  BRAVE_HTML_CANVAS_ELEMENT_H_
  protected:
   void DidMoveToNewDocument(Document& old_document) override;
   void DidRecalcStyle(const StyleRecalcChange change) override;
