diff --git a/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc b/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc
index 22b3385ca37a6a16c8ba5a7d9b1807b04575347f..5d417c35c867faa8931cb79bc245739dd8660181 100644
--- a/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc
+++ b/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc
@@ -396,6 +396,7 @@ bool ClipboardCommands::ExecuteCopy(LocalFrame& frame,
       frame.Selection().IsHidden())
     return true;
 
+  BRAVE_CLIPBOARD_COMMANDS_CHECK_SOURCE
   if (EnclosingTextControl(
           frame.Selection().ComputeVisibleSelectionInDomTree().Start())) {
     frame.GetSystemClipboard()->WritePlainText(frame.SelectedTextForClipboard(),
