diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index 3e2993e44729f7e9866609eb0af6e4795b911d74..b78c79e35b4dd7e3be37fbe30f29502cf0851bdd 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -372,5 +372,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
