diff --git a/scripts/build/typescript/typescript.gni b/scripts/build/typescript/typescript.gni
index 022ac763843b60ff0621231d9202e6b0edc9b4f0..6ed6e96adf9500f19d27827a255e7abd2ca5c31e 100644
--- a/scripts/build/typescript/typescript.gni
+++ b/scripts/build/typescript/typescript.gni
@@ -200,6 +200,7 @@ template("ts_library") {
 
       foreach(src, sources) {
         _extension = get_path_info(src, "extension")
+        if (label_matches(src, ["//brave/*"])) { src = get_path_info(src, "name") }
 
         # Any .d.ts file does not generate a corresponding a .js file, but we
         # copy it over into the generated build folder instead, as tsc doesn't
