diff --git a/BUILD.gn b/BUILD.gn
index e3e38a956ced6f4a00e118159e40fa1f220c5c4e..6a33c8e06cd5c634083a02585914e4ccbfe7118a 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -903,6 +903,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3583,6 +3584,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
